numpy/cppbuild.sh

```diff
@@ -85,6 +85,11 @@ fi
 export PYTHONPATH="$PYTHON_INSTALL_PATH"
 mkdir -p "$PYTHON_INSTALL_PATH"
 
+# Install pip for the cross-compiled interpreter
+if ! "$PYTHON_BIN_PATH" -m pip --version >/dev/null 2>&1; then
+    "$PYTHON_BIN_PATH" -m ensurepip --upgrade
+fi
+
```

Comment:
The missing pip in the CPython package is probably caused  by getting it built in cross-compilation mode. Try to update the linux-arm64 as per the old linux-armhf patterns in cppbuild.sh to build in native mode

Reply:
thanks, I'll test that out locally first

Reply:
Can we remove this now?

Reply:
It seems like it is still required. When I remove it, we get the original error - 
`/home/runner/.javacpp/cache/cpython-3.13.5-1.5.13-SNAPSHOT-linux-arm64.jar/org/bytedeco/cpython/linux-arm64/bin/python3.13: No module named pip`
I assume its now picking up the natively compiled cpython snapshot that ran [6 hours ago](https://github.com/bytedeco/javacpp-presets/actions/runs/16479122470/job/46603356679#step:2:9278) -